fp.md

```diff
@@ -14,7 +14,7 @@ people.forEach(function(person){
 });
 ```
 ```js
-// Your answer goes here
+var ageArray = people.age.map[];
```

Comment:
Take a look through the `.map` documentation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map?v=example

Let me know if you have any questions on how/when `.map` should be used. Happy to talk about it.